src/planner.ts

```diff
@@ -61,24 +61,63 @@ export function geometryToGeoJSON(
   };
 }
 
+const entranceQuery = `
+[out:json][timeout:25];
+(
+  relation(around:10, LAT, LON)[building];
+  way(r);
+  way(around:10, LAT, LON)[building];
+)->.b;
+// gather results
+(
+  node(w.b)[entrance];
+);
+// print results
+out body;
+>;
+out skel qt;
+`;
+
 export default function calculatePlan(
   origin: [number, number],
   destination: [number, number],
   callback: (f: FeatureCollection) => void
 ): void {
-  const planner = new FlexibleTransitPlanner();
-  planner
-    .query({
-      from: { latitude: origin[0], longitude: origin[1] },
-      to: { latitude: destination[0], longitude: destination[1] },
-      roadNetworkOnly: true
+  const url = new URL("https://overpass-api.de/api/interpreter");
+  url.searchParams.append(
+    "data",
+    entranceQuery
+      .replace(/LAT/g, destination[0].toString())
+      .replace(/LON/g, destination[1].toString())
```

Comment:
Consider a function instead, something like this:

```js
const buildEntranceQuery = (lat, lon) => {
  return `foo
    ${lat} bar ${lon}
    baz`
}
buildEntranceQuery(lat, lon);
```

Reply:
Implemented.